src/userscores/internal/exportscorescenario.cpp

```diff
@@ -223,7 +253,8 @@ bool ExportScoreScenario::askForRetry(const QString& filename) const
     return btn == static_cast<int>(IInteractive::Button::Retry);
 }
 
-bool ExportScoreScenario::doExport(INotationWriterPtr writer, const INotationPtrList& notations, const io::path& path,
+template<typename N>
```

Comment:
Please, remove the template for this method, this is confused
Divide the method in two if needed